src/icon-manifest/supportedExtensions.ts

```diff
@@ -488,6 +488,7 @@ export const extensions: IFileCollection = {
     { icon: 'svg', extensions: ['svg'], format: FileFormat.svg },
     { icon: 'swagger', extensions: [], languages: [languages.swagger], format: FileFormat.svg },
     { icon: 'swift', extensions: ['package.pins'], filename: true, languages: [languages.swift], format: FileFormat.svg },
+    { icon: 'swig', extensions: [], languages: [languages.swig], format: FileFormat.svg },
```

Comment:
where's the definition of that language?

Reply:
Already in.

Reply:
How could we missed that?

Reply:
Missed?

Reply:
I mean like: Why do we defined the language without the icon and the entry in our collection?

Reply:
Cause there was no icon at that time. I just found the time now to create an icon for it.

Reply:
Ah, sounds reasonable. Thank you!